src/tactics/GameState.js

```diff
@@ -291,7 +291,7 @@ export default class GameState {
           type: 'ChaosSeed',
           assignment: [5, 5],
         }],
-        position: 'C',
+        position: 'N',
```

Comment:
This doesn't look right.  While it should work, the egg is not in the North side of the board.  Rather, I think the following code does not properly support the C value (center)